### ISSUE TITLE
Baseline: image classification pipeline

### ISSUE THREAD

**Issue by ArchebasovVl** [ArchebasovVl/Distributed-ML#1]
Создание бейзлайна классификатора может быть полезно для:
* формирования пайплайна обучения модели;
* измерения основных метрик для сравнения с последующими версиями модели.

**Предложения:**

* Выбрать архитектуру модели ([ResNet](https://pytorch.org/hub/pytorch_vision_resnet/), [EfficientNet](https://huggingface.co/docs/transformers/en/model_doc/efficientnet), [MobileNet](https://huggingface.co/learn/computer-vision-course/en/unit2/cnns/mobilenet), [VisionTransformer](https://huggingface.co/docs/transformers/en/model_doc/vit), [ConvNeXT](https://huggingface.co/docs/transformers/en/model_doc/convnext) или другие)
* Выбрать датасет ([CIFAR-10 / CIFAR-100](https://pytorch.org/vision/0.19/generated/torchvision.datasets.CIFAR10.html), [ImageNet](https://www.image-net.org/), [MNIST / Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) или другие)
* Реализовать пайплайн обучения на одном устройстве (по возможности в виде реализации Mixin-класса фреймворка наподобие [scikit-learn.pipeline](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html))
* Сформировать отчет, содержащий метрики: (Accuracy, Precision, Recall, F1-score, ROC-AUC, время обучения)
* Разместить результат работы в `models/baseline/`

**Дальнейшая работа:** 
После выполнения предложенных задач возможно маштабирование пайплайна на несколько устройств

Все предложения доступны для обсуждения и корректировки
